### DISCUSSION TITLE
Can embedding be done in real time during note creation?

### DISCUSSION THREAD

**Discussion by is-uko** [brianpetro/obsidian-smart-connections#851] [Q&A]
Do Smart Connections results change in real time during note creation?

Also, on a new note, I get "No entry found for key" and have to restart to see the results.

**Comment by brianpetro** [comment]
Realtime embedding is being rolled out in the Smart Connect app (exists in the latest version). It will still take a bit of time to get the logic up to speed in the Smart Connections Obsidian plugin so that it utilizes those embeddings efficiently.

In the meantime, the "Refresh & make connections" command will update embeddings for the current note 🌴

**Comment by is-uko** [comment]
I am on version 2.2.83, and in a new note I get "No entry found for key".
